### ISSUE TITLE
[Accessibility]A11y_VSCode_UserSettings_Keyboard: Keyboard focus order is improper when Tab or Shift Tab key is used while navigating through the elements(list items) present in the pane i.e.(Commonly Used,Editor,Workbench………..etc)

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52470]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : HighSierra10.13.5

**Additional Details:**
MAS Violated : MAS 2.4.3

**Repro Steps:**
1.Launch VS Code.
2.Navigate to Settings icon in Activity Bar->Press Enter->Select "Settings" from dropdown.
3.Keyboard focus order is improper when Tab or Shift Tab key is used while navigating through the elements(list items) present in the pane.

**Actual:**
Keyboard focus order is improper when Tab or Shift Tab key is used while navigating through the elements(list items) present in the pane.

When the focus goes to the topmost element of the pane and then Tab key is pressed then the focus goes abruptly shifts to some other controls of the screen.

Similarly,when the focus goes to the last element of the pane and then Tab key is pressed then the focus goes abruptly shifts to some other controls of the screen

**Expected:**
Keyboard focus order is improper when Tab or Shift Tab key is used while navigating through the elements(list items) present in the pane.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The End user will not be able to access the edit buttons which is embedded with some functionalities.

**MAS Reference:**
MAs 2.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment For Reference:**
[A11y_VSCode_UserSettings_Keyboard_FocusOrderAbrupt.pptx](https://github.com/Microsoft/vscode/files/2119285/A11y_VSCode_UserSettings_Keyboard_FocusOrderAbrupt.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52467
